pandunia-lekse.csv

```diff
@@ -1782,6 +1802,7 @@ melovespe||||honeybee||abeja||||||||||||||mehiläinen|pszczoła miodna
 mema||||remember (recall)|se rappeler (se remémorer)|acordarse|lembrar-se (recordar-se)|помнить|تَذَكَّرَ|记得 (回忆)|思い出す|기억하다|nhớ|याद करना|মনে রাখা|ingat||hatırlamak|memori|muistaa|pamiętać
 memba||||belong (be a member)||pertenecer (ser miembro de)||||||||||||||kuulua (olla jäsenenä)|należeć (być członkiem)
 member||||member (component)||miembro (componente)|||||||||||||membro|jäsen (osatekijä, komponentti)|członek
+membrane|||eng:fra:membrane, spa:por:ita:membrana, deu:Membran, rus:мембрана (membrana), may:membran|membrane (film)|membrane|membrana|membrana|плева (перепонка)||膜|膜||||||||||
```

Comment:
eska mome xudu muta mene da "filme"?

Reply:
sa multi bax, "filme" mena "kinografe" e no mena "faini warke". we saba, me teroru le abla galto ahu.
lekino, ablo we no mal. "film" e "movie" si samonam sa engli e espani, e mome no konfuzu lole. duli lekse da mena "movie" okey.